libfreerdp/core/server.c

```diff
@@ -295,6 +300,7 @@ static void wts_read_drdynvc_pdu(rdpPeerChannel* channel)
 	{
 		WLog_ERR(TAG,  "received Cmd %d but channel is not ready.", Cmd);
 	}
+	return TRUE;
```

Comment:
Note (nothing todo with you pull request):
In case the command is unrecognized we would just return TRUE and indicate success but 
if the cmd isn't know it might be an protocol problem. Maybe we should return FALSE here.
@awakecoding what do you think on that?